### DISCUSSION TITLE
TextBufferGeometry exists in TextGeometry.d.ts, but not in the JavaScript [@types/three]

### DISCUSSION THREAD

**Discussion by TradeIdeasPhilip** [DefinitelyTyped/DefinitelyTyped#60129] [Issues with a @types package]
Here is the offending line:  https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/three/examples/jsm/geometries/TextGeometry.d.ts#L38

`export { TextGeometry as TextBufferGeometry };`

There is no corresponding line anywhere in [the original code](https://github.com/mrdoob/three.js).

There _is_ a `TextGeometry` class in both the JavaScript and the .d.ts.  It is only exported as `TextGeometry`, not as `TextBufferGeometry`.

[a comment by an automated account is not shown]

**Comment by joshuaellis** [comment]
Bugs should be written up in the three-ts-types repo please so we can address them! :)
